Makefile

```diff
@@ -33,16 +33,20 @@ typelint: install-mypy ## Runs type linting
 ansible-config-lint: ## Runs custom Ansible env linting tasks.
 	molecule verify -s ansible-config
 
+.PHONY: docs
+docs: ## Build project documentation in live reload for editing
+# Spins up livereload environment for editing; blocks.
+	make -C docs/ clean && sphinx-autobuild docs/ docs/_build/html
+
 .PHONY: docs-lint
 docs-lint: ## Check documentation for common syntax errors.
 # The `-W` option converts warnings to errors.
 # The `-n` option enables "nit-picky" mode.
 	make -C docs/ clean && sphinx-build -Wn docs/ docs/_build/html
 
-.PHONY: docs
-docs: ## Build project documentation in live reload for editing
-# Spins up livereload environment for editing; blocks.
-	make -C docs/ clean && sphinx-autobuild docs/ docs/_build/html
+.PHONY: docs-linkcheck
```

Comment:
can we add a note about this makefile target to our docs on... working on the docs? (meta) 

and let's also run this in CI: you can do this by adding another run step that runs `make docs-linkcheck` in our circle ci config file [here](https://github.com/freedomofpress/securedrop/blob/develop/.circleci/config.yml#L72)

Reply:
Thanks! Force pushed those changes.

Reply:
Is this the right approach?

https://github.com/freedomofpress/securedrop/pull/4487/files#diff-1d37e48f9ceff6d8030570cd36286a61R74